API-Samples/utils/util_init.cpp

```diff
@@ -1757,10 +1798,9 @@ void init_image(struct sample_info &info, texture_object &texObj, const char *te
     image_create_info.mipLevels = 1;
     image_create_info.arrayLayers = 1;
     image_create_info.samples = NUM_SAMPLES;
-    image_create_info.tiling = VK_IMAGE_TILING_LINEAR;
+    image_create_info.tiling = texObj.needs_staging ? VK_IMAGE_TILING_OPTIMAL : VK_IMAGE_TILING_LINEAR;
     image_create_info.initialLayout = VK_IMAGE_LAYOUT_PREINITIALIZED;
```

Comment:
UNDEFINED if not linear (I could see this becoming a VU -- PREINIT can only be used for linear...